CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,46 @@
+# Community Code of Conduct
+
+**Version 1.2  
+August 19, 2020**
```

Comment:
This is a outdated version of the CoD file. Please use current version, which is available [here](https://github.com/eclipse/.github/blob/master/CODE_OF_CONDUCT.md?plain=1).